hevc/sps.go

```diff
@@ -181,3 +183,73 @@ func (s *SPS) ImageSize() (width, height uint32) {
 	height = encHeight - (s.ConformanceWindow.TopOffset+s.ConformanceWindow.BottomOffset)*subHeightC
 	return width, height
 }
+
+// CodecString returns string based on SPS fields.
+// ISO/IEC 14496-15:2014 Annex E.
+func (s *SPS) CodecString() string {
+
+	fields := []string{"hvc1"}
+	generalProfileSpace := ""
```

Comment:
"hvc1" is too hard-coded. One may want to either have "hvc1" or "hev1" here depending on whether the parameters are out-of-band or in-band.